pkgs/development/python-modules/jaxlib/default.nix

```diff
@@ -48,9 +51,10 @@ let
   inherit (cudaPackages) cudatoolkit cudnn nccl;
 
   pname = "jaxlib";
-  version = "0.3.0";
+  version = "0.3.15";
 
   meta = with lib; {
+    broken = stdenv.isDarwin;
```

Comment:
broke goes last in meta. It being first was an automated accident.

Reply:
Fixed by rebase